File: csv/fsharp/update/csv_update.fs
// ----------------------------------------------------------------
//	csv_update.cs
//
//						Aug/06/2011
//
// ----------------------------------------------------------------
open System
open System.Text
open Microsoft.FSharp.Core
open System.IO

// ----------------------------------------------------------------
[<EntryPoint>]
let main (args : string[] ) =

  printfn "*** 開始 ***"

  let file_txt = args.[0]

  let id_in = args.[1]
  let population_in = int args.[2]
  printfn "%s\t%d" id_in population_in

  let dict_aa = text_manipulate.csv_read_proc (file_txt)

  let dict_aa = text_manipulate.dict_update_proc (dict_aa,id_in,population_in)

  let _ = text_manipulate.dict_display_proc (dict_aa)

  let _ = text_manipulate.csv_write_proc (file_txt,dict_aa)

  printfn  "*** 終了 ***"
  0
// ----------------------------------------------------------------

File: csv/go/update/csv_update.go
// ----------------------------------------------------------------
//
//	csv_update.go
//
//					May/20/2018
//
// ----------------------------------------------------------------
package main

import (
	"fmt"
	"os"
	"strconv"
)

// ----------------------------------------------------------------
func main() {
	fmt.Printf ("*** 開始 ***\n")

	csv_file := os.Args[1]
	id_in := os.Args[2]
	population_in,_ := strconv.Atoi (os.Args[3])

	fmt.Printf ("id_in = %s\t" , id_in)
	fmt.Printf ("population_in = %d\n" , population_in)

	var dict_aa = csv_read_proc (csv_file)

	dict_aa = dict_update_proc (dict_aa,id_in,population_in )

	csv_write_proc (csv_file,dict_aa)

	fmt.Printf ("*** 終了 ***\n")
}

// ----------------------------------------------------------------
